internal/util/fsql/metrics.go

```diff
@@ -71,8 +71,65 @@ func (c *metricsCollector) Collect(ch chan<- prometheus.Metric) {
 		float64(stats.OpenConnections),
 	)
 
-	// Add other metrics from stats here.
-	// TODO https://github.com/FerretDB/FerretDB/issues/2909
+	ch <- prometheus.MustNewConstMetric(
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "in_use"),
+			"The number of connections currently in use.",
+			nil, c.labels,
+		),
+		prometheus.GaugeValue,
+		float64(stats.InUse),
+	)
+
+	ch <- prometheus.MustNewConstMetric(
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "idle"),
+			"The number of idle connections.",
+			nil, c.labels,
+		),
+		prometheus.GaugeValue,
+		float64(stats.Idle),
+	)
+
+	ch <- prometheus.MustNewConstMetric(
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystem, "wait_count"),
```

Comment:
After reading https://prometheus.io/docs/practices/naming/, I think that all counters should have a `_total` suffix (but the wording surely could be clearer). WDYT?

Reply:
I agree, with `_total` suffix, I think, it's easy to understand what is a type of this metrics(counter), without looking into a code, and starting to apply `rate()` query functions or etc.